platform/ABModelQuery.js

```diff
@@ -1,3 +1,4 @@
+const _ = require("lodash");
```

Comment:
I think on the ab_platform_service the `ABFactory` already includes the `_.cloneDeep` method exposed.